apiary.apib

```diff
@@ -988,3 +988,26 @@ Fetches a bundle by identifier
     Bundle not found
 
     + Body
+
+## Group Nodeinfo
+
+Information on the Gateway node.
+
+## /nodeinfo
+
+#### Get node info [GET]
+
++ Response 200
+
+    + Attributes (object)
+        + commit (string) - Hash of the latest git commit on the node
+        + version (string) - Version of AMB-NET
```

Comment:
Version of AMB-NET??? Shouldn't it rather be Ambrosus Node?

Reply:
true, it should be the version of the node